contracts/project/Builders.sol

```diff
@@ -12,26 +12,147 @@ contract Builders {
         bytes32 publicKey;
     }
 
+    struct BuilderJoinRequest {
+        address _address;
```

Comment:
I would maybe remove the underscore. I know we have it in Data provider structure, but when I work with it in JS, ESlint was shouting at me for that `_` in the beginning 😄